packages/@glimmer/validator/index.ts

```diff
@@ -39,12 +39,17 @@ export {
   consumeTag,
   isTracking,
   track,
-  trackedData,
   memo,
   untrack,
   isConstMemo,
+  Cache,
```

Comment:
Should this be exported? I thought the idea was that you'd always have to go through our accessors 🤔